Rakefile

```diff
@@ -135,38 +146,37 @@ end
 namespace :info do
   desc 'Show Delivery admin credentials'
   task :delivery_creds do
-    # TODO - use Ruby to read these files so its cross-platform
-    system "cat .chef/delivery-cluster-data/*.creds"
+    # TODO: Use Ruby to read these files so its cross-platform
+    system 'cat .chef/delivery-cluster-data/*.creds'
   end
 
   desc 'List all your core services'
   task :list_core_services do
-    system "knife search node 'name:*server* OR name:build-node*' -a ipaddress"
-    system "grep chef_server_url .chef/delivery-cluster-data/knife.rb"
+    system 'knife search node \'name:*server* OR name:build-node*\' -a ipaddress'
```

Comment:
This is one where switching to single quotes seems weird to me cause now we have to escape. I think for consisstency I want ingle quotes and escapes, but for readability I kinda want doubles on this.